src/js/components/RuleLabel.react.js

```diff
@@ -0,0 +1,42 @@
+import type { Props } from '../containers/AppContainer.react';
+import { RulePropertyUtils } from '../utils/RulePropertyUtils';
+
+/**
+ * Copyright 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+const React = require('react');
+const level_1 = '•';
+const level_2 = '✘';
+const level_3 = '✔';
+
+class RuleLabel extends React.Component<Props, State> {
+  render() {
+    return (
+      <div>
+        {this.props.className && this.props.htmlFor
+          ? '<label className="' +
+            this.props.className +
+            '" htmlFor="' +
+            this.props.htmlFor +
+            '">'
+          : '<label>'}
```

Comment:
This will simply output the string `'<label>'` instead of the `<label>` element. Please remove the single quotes.

Also, and more importantly, please test your result with `npm start` before submitting a Pull Request, as this error is very apparent on the UI:

<img width="316" alt="screen shot 2018-04-13 at 2 45 21 pm" src="https://user-images.githubusercontent.com/1878108/38750645-f9c95dd0-3f2b-11e8-9730-867e836b89db.png">